doc/userguide/rules/enip-keyword.rst

```diff
@@ -1,40 +1,225 @@
 ENIP/CIP Keywords
 =================
 
-The enip_command and cip_service keywords can be used for matching on various properties of
-ENIP requests.
+enip_command
+------------
 
-There are three ways of using this keyword:
+For the ENIP command, we are matching against the command field found in the ENIP encapsulation.
+
+Examples::
 
-* matching on ENIP command with the setting "enip_command";
-* matching on CIP Service with the setting "cip_service".
-* matching both the ENIP command and the CIP Service with "enip_command" and "cip_service" together
+  enip_command:99;
+  enip_command:ListIdentity;
 
 
-For the ENIP command, we are matching against the command field found in the ENIP encapsulation.
+cip_service
+-----------
 
 For the CIP Service, we use a maximum of 3 comma separated values representing the Service, Class and Attribute.
 These values are described in the CIP specification. CIP Classes are associated with their Service, and CIP Attributes
 are associated with their Service. If you only need to match up until the Service, then only provide the Service value.
 If you want to match to the CIP Attribute, then you must provide all 3 values.
 
-
-Syntax::
-
-  enip_command:<value>
-  cip_service:<value(s)>
-  enip_command:<value>, cip_service:<value(s)>
-
-
 Examples::
 
-  enip_command:99
   cip_service:75
   cip_service:16,246,6
-  enip_command:111, cip_service:5
 
 
 (cf. http://read.pudn.com/downloads166/ebook/763211/EIP-CIP-V1-1.0.pdf)
 
 Information on the protocol can be found here:
 `<http://literature.rockwellautomation.com/idc/groups/literature/documents/wp/enet-wp001_-en-p.pdf>`_
+
+enip.status
+-----------
+
+For the ENIP status, we are matching against the status field found in the ENIP encapsulation.
+It uses a 32-bit unsigned integer as value.
+
+Examples::
+
+  enip.status:100;
+  enip.status:>106;
```

Comment:
For all these cases where a value is accepted, are all other modes (<, <=, >...) also accepted? Wonder if this should be made more evident, if nothing else, by using other operands, too.

Reply:
Every mode is supported is true for all integer keywords...

Maybe there should be from every integer keyword documentation a link to some special section where integers are documented ? This should be its own redmine ticket ;-)

Reply:
Fair point, and makes sense.

Reply:
https://redmine.openinfosecfoundation.org/issues/6628